src/pages/steps/AdditionalInformationStep/FormScore.tsx

```diff
@@ -184,6 +136,56 @@ const getMinimumScore = (weights: Weights): number => {
   return minimumScore;
 };
 
+const DynamicBarometerIcon = ({ minimumScore, score, size = 70 }) => (
+  <div
+    style={{
+      width: size,
+      height: size,
+      position: 'relative',
+    }}
```

Comment:
Why aren't you using styled component styles? Same for the rest of the styles applied below